[styles] Add Bowling Alley

endim8 opened the pull request:
icon day
<img width="32" src="https://user-images.githubusercontent.com/26939824/209444951-55c5e37b-9c63-49f9-8e16-496c2538ee2c.svg"/>
icon night:
<img width="32" src="https://user-images.githubusercontent.com/26939824/209444960-5913b8c7-00e1-4208-bd02-724e082e39d4.svg"/>
(made by me so CC0)

<img width="764" alt="Screenshot 2022-12-24 at 16 52 23" src="https://user-images.githubusercontent.com/26939824/209445172-2c12a627-5eef-42ea-ab4c-e144cb61e839.png">

Fixes https://github.com/organicmaps/organicmaps/issues/4144

vng commented:
We already have bowling-m for types sport=9pin, sport=10pin.
I think we can/should combine all this types in one icon, no?

endim8 commented:
IMO the old icon doesn't look very good:
<img width="42" src="https://user-images.githubusercontent.com/26939824/210408628-0adb5a86-7ef6-4890-b973-c1fc154cb969.svg"/>

the latest push removes the old icons & drawing rules, but i can switch to the old one if wanted :)

vng commented:
- leisure|bowling_alley;255; is ok, but please, put it somewhere below 255. According to the usage, please keep 255 for more "often" type.
- Well, sport=9pin/10pin maybe a _secondary_ type, but why not to draw specific icon? Like we do for football/basketball/etc ... Other question, is it ok with current POI icon? I think its ok. Or worth to revert back black/white bawling icon?

vng commented:
@j13m126 
https://github.com/organicmaps/organicmaps/pull/2960

endim8 commented:
> leisure|bowling_alley;255; is ok, but please, put it somewhere below 255.

Will move it lower 👍
> sport=9pin/10pin maybe a secondary type, but why not to draw specific icon? Like we do for football/basketball/etc

it seems there are three ways bowling is mapped:
- as a primary feature eg. https://www.osm.org/node/4767656648
- as a secondary feature (only represented as tags) eg. https://www.osm.org/node/4588495941
- as a secondary feature (as it's own node) eg. https://www.osm.org/node/2234544743

In the first case, the new icon is good.
In the second case, the primary feature icon (pub, leisure_centre, restaurant) should be displayed instead of the bowling icon
in the third case, maybe using the old black and white icon is good, but I'm not sure how useful it would be.

vng commented on data/strings/types_strings.txt:
```diff
@@ -1118,6 +1118,46 @@
+    pt = Pista de boliche
+    pt-BR = Pista de boliche
+    ro = Popicărie
+    ru = Кегельбан
```
ru = Боулинг

vng commented on data/categories.txt:
```diff
@@ -4185,6 +4185,46 @@ sk:3Kino
+pt:Pista de boliche
+pt-BR:Pista de boliche
+ro:Popicărie
+ru:Кегельбан
```
ru:4Боулинг

vng reviewed (commented):
Ok, I vote for revert back sport=9/10pin black/white icons.
For leisure - use separate new (POI style) icon.

vng commented:
And yes, we should check draw priorities (e g 9pin icon should NOT replace restaurant or leisure icon). I can add tests later.

endim8 commented:
![Screenshot_20230105-150015_Organic Maps](https://user-images.githubusercontent.com/26939824/210811196-c5165819-07d1-4f17-acd8-8a5ac21064ad.png)


https://www.osm.org/way/28883091

The sport=9/10pin type seems to have a higher priority than bowling alley (this poi has 'leisure=bowling_alley' & 'sport=10pin')

endim8 commented:
fixed type priority
![signal-2023-01-05-163757_005](https://user-images.githubusercontent.com/26939824/210833459-53412260-baff-42fe-bdf0-32bb479e2d61.png)

and added correct ru translations 👍

endim8 commented:
finally good to merge :)

matheusgomesms commented on data/categories.txt:
```diff
@@ -4185,6 +4185,46 @@ sk:3Kino
+nb:Bowlinghall
+nl:Bowlingbaan
+pl:Kręgielnia
+pt:Pista de boliche
```
pt:Pista de bowling

endim8 replied:
Should `pt-BR:Pista de boliche` be changed to bowling as well?

matheusgomesms replied:
No, just PT. PT-BR is already correct (Pista de boliche), thanks!

endim8 replied:
cool, thank you for the translations :)

Kazing commented on data/strings/types_strings.txt:
```diff
@@ -1118,6 +1118,46 @@
+    eu = Bolatokia
+    fa = ﮓﻨﯿﻟﻮﺑ ﻦﻟﺎﺳ
+    fi = Keilarata
+    fr = Allée de bowling
```
fr:Salle de bowling

Kazing reviewed (changes requested):
A little change in fr

endim8 commented:
> A little change in fr

thank you :) updated
